packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -54,31 +71,69 @@ export function CrossmintWalletProvider({
             return { startedCreation: false, reason: `Jwt not set in "CrossmintProvider".` };
         }
 
-        const internalCall = async () => {
+        if (enablePasskeyPrompt && config.signer && "type" in config.signer && config.signer.type === "PASSKEY") {
+            if (config.onCreateWalletPasskeyCallback == null) {
+                config.onCreateWalletPasskeyCallback = () =>
+                    new Promise<void>((resolve) => {
+                        setPasskeyPromptState({
+                            type: "create-wallet",
+                            open: true,
+                            primaryActionOnClick: () => {
```

Comment:
What if they close the modal?

Reply:
I haven't handled closing the modal yet because it's not clear when they'll be able to close it. I was gonna make that a follow up once I have more defined requirements around dismissible modals